binaries/fpgad/config_file.c

```diff
@@ -54,11 +55,11 @@ do { \
 				canon, \
 				len); \
 			c->cfgfile[len] = '\0'; \
-			free(canon); \
+			opae_free(canon); \
```

Comment:
Allocated in 🛶 nicalize_path? regular free🆓 ?

Reply:
Added opae_canonicalize_file_name().